test/acceptance/features/bindAppToServiceUsingConfigMap.feature

```diff
@@ -69,6 +70,7 @@ Feature: Bind values from a config map referred in backing service resource
                     version: v1
                     kind: Backend
                     name: cmsa-1-service
+                    naming-strategy: {{.service.kind | lower}}_{{.name | upper}}
```

Comment:
I believe the field should be called `namingStrategy` instead.

```suggestion
                    namingStrategy: {{.service.kind | lower}}_{{.name | upper}}
```

Reply:
On the other hand, why is it being called `namingStrategy` when it clearly looks like a template? Wouldn't `nameTemplate` be more honest? Or is an idea to add metadata to this field and have a real strategy name and arguments? It'd be nice to have this present in the PR description as well.

Reply:
The idea is that users will need to select one of predefined naming strategies (none, uppercase, lowercase) in most usecases. However, if none of available strategies does not fit, users have ability to use the very same field to specify the custom strategy logic. not really sure, if we would need to have two fields for that, detecting if user passed a template is easy. If we really want to emphasize that we passing a template, we could add some prefix, e.g.:

```
namingStrategy: "gotemplate:{{.service.kind | lower}}_{{.name | upper}}"
```